overlord/snapstate/snapstate.go

```diff
@@ -1168,7 +1168,7 @@ func InstallPathMany(ctx context.Context, st *state.State, sideInfos []*snap.Sid
 
 // InstallMany installs everything from the given list of names.
 // Note that the state must be locked by the caller.
-func InstallMany(st *state.State, names []string, userID int) ([]string, []*state.TaskSet, error) {
+func InstallMany(st *state.State, names []string, userID int, transactional bool) ([]string, []*state.TaskSet, error) {
```

Comment:
Hm would it be possible to split the whole PR into 3 bits? like so;
1. extend InstallMany/InstallPathMany/RemoveMany/UpdateMany with extra argument, leave a TODO In each, update tests to pass flag indicating no transaction (so mostly code churn)
2. update client/daemon code to pass the right flags
3. update snapstate code to behave according to the flag, extend unit tests, add managers tests